core/src/KokkosExp_MDRangePolicy.hpp

```diff
@@ -53,7 +53,9 @@
 #include <Kokkos_Parallel.hpp>
 
 #if defined(__CUDACC__) && defined(KOKKOS_ENABLE_CUDA)
+#ifdef KOKKOS_ENABLE_DEPRECATED_CODE
 #include <Cuda/KokkosExp_Cuda_IterateTile.hpp>
+#endif
```

Comment:
Should it be an `#else`?  Had we forgotten to mark it has deprecated?

Reply:
no, it's used in deprecated code, so I just left it as is.